src/parser/jobs/smn/modules/Summons.tsx

```diff
@@ -313,31 +313,57 @@ export class Summons extends Analyser {
 				key: summon.start,
 				title: {
 					content: <>
-						{this.parser.formatEpochTimestamp(summon.start)}: <ActionLink {...this.data.getAction(summon.data.demiSummon.action)} />
+						{<span style={{ width: '200px', display: 'inline-block' }}>{this.parser.formatEpochTimestamp(summon.start)}: <ActionLink {...this.data.getAction(summon.data.demiSummon.action)} /></span>}
```

Comment:
Fixed widths like this are tricky with international sites like xiva.  (Both Japanese and German break in this case.)  I think we will need to either accept the egi icons not lining up or right-aligning them.

Given that this is a relatively fixed list of strings and translations, it's tempting to just size to the largest string, but I'm not sure that will hold up when other browsers, personal settings, and mobile gets involved.

Reply:
I'm not sure which size would work best, so if you can find a size that fits all that would be awesome. As long as the size is small enough it should be good for mobile as well I think? Maybe there is an align char that we can use?

Reply:
I don't think there is a size that works best.  I only have basic web layout knowledge, so I'm not sure what kind of tricks are available to force alignments.  It's functional, if not as aesthetically pleasing, to just have them not line up vertically across rows.